docs/guide/database-access.md

```diff
@@ -0,0 +1,59 @@
+# Accessing the database directly
+
+## Working with collections
+As stated earlier, the main goal of the Apostrophe JavaScript content API is to facilitate requests to the MongoDB database without the developer having to know advanced MongoDB syntax or worry about user permissions. However, there are instances where you might want to directly access and modify your collections without the overhead or restrictions of the Apostrophe model layer.
```

Comment:
That's good, and link the words "Apostrophe model layer" to the article that covers queries, insert and update using our standard methods.

Reply:
Added.